### ISSUE TITLE
Columns missing backticks

### ISSUE THREAD

**Issue by leovanhaaren** [Raizlabs/DBFlow#1082]
DBFlow Version: **4.0.0-beta2**
Issue Kind: **Bug**

Description: 
`Error Code : 1 (SQLITE_ERROR)
Caused By : SQL(query) error or missing database.

 (near "order": syntax error (code 1): , while compiling: UPDATE  OR ABORT `CollectionEntity` SET order=?,country=?,image_url=?,type=?,root_order=?,id=?,ratio=? WHERE `id`=1)`

Why are the "`" (backticks) missing on columns here?

**Comment by agrosner** [comment]
Can you send the actual DBFlow code you're calling here? thank you.

**Comment by DreierF** [comment]
I've had the same problem, because I used the keyword `index` as column name. Here's a stacktrace:
```
android.database.sqlite.SQLiteException: near "index": syntax error (code 1): , while compiling: UPDATE  OR ABORT `PASSE` SET exact=?,round=?,index=?,_id=?,save_time=?,image=? WHERE `_id`=1
at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:889)
at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:500)
at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
at android.database.sqlite.SQLiteDatabase.updateWithOnConflict(SQLiteDatabase.java:1577)
at com.raizlabs.android.dbflow.structure.database.AndroidDatabase.updateWithOnConflict(AndroidDatabase.java:71)
at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.update(ModelSaver.java:72)
at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.save(ModelSaver.java:44)
at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.save(ModelSaver.java:31)
at com.raizlabs.android.dbflow.structure.ModelAdapter.save(ModelAdapter.java:102)
at de.dreier.mytargets.shared.models.db.End_Table.save(End_Table.java:267)
at de.dreier.mytargets.shared.models.db.End_Table.save(End_Table.java:34)
at com.raizlabs.android.dbflow.structure.BaseModel.save(BaseModel.java:62)
```

This in turn leads to this line in AndroidDatabase.updateWithOnConflict:
```
count = database.updateWithOnConflict(tableName, contentValues, where, whereArgs, conflictAlgorithm);
```

So the bug comes from the android SQLite implementation itself!

**Comment by agrosner** [comment]
yeah it looks the the `ContentValues` are not backticked in generated code. Fixed in `develop` for `4.0.0-beta3`
